apps/meteor/app/lib/server/startup/settings.ts

```diff
@@ -3217,6 +3217,15 @@ settingsRegistry.addGroup('Call_Center', function () {
 					value: true,
 				},
 			});
+			this.add('VoIP_Enable_Keep_Alive_For_Flaky_Networks', false, {
```

Comment:
Maybe instead of "flaky", we can use "unstable"? 👀

Reply:
Fixed.